### DISCUSSION TITLE
running seurat in parallel on multiple nodes

### DISCUSSION THREAD

**Discussion by ktyssowski** [satijalab/seurat#4693] [Q&A]
in looking through the future package documentation, i noticed that it is possible to run on multiple nodes (rather than just multiple cores) using the plan(cluster) setting. i am wondering if this is compatible with seurat (i.e., can i run plan(cluster) instead of plan(multiprocess) or plan(multicore) before finding markers or integration anchors etc)?

**Comment by timoast** [comment]
Yes it is, you can use any settings in the `future` package
